### PR TITLE
Rename-admin

### PR DESCRIPTION
**Pull request by little9**
This renames the Drupal admin user via a drush sql query.

Related to iDC-general#453

### CODE REVIEW

**Comment by little9** [comment]
@jhujasonw Does the username be a secret? I think the purpose of changing the admin username is to 
avoid basic Drupal hacks/scripts that assume the admin user is named `admin`. This would accomplish that.

File: drupal/rootfs/etc/cont-init.d/99-custom-setup.sh [hunk 1]
```diff
@@ -142,8 +142,12 @@ function main {
     exit 1
   fi
 
+
+  # Rename the default drupal admin user
+  drush sql-query "UPDATE users_field_data SET name='jhu_admin' WHERE uid=1";
+
   # Set the admin password based on env vars
-  drush -l "${site_url}" user-password admin "${DRUPAL_DEFAULT_ACCOUNT_PASSWORD}";
+  drush -l "${site_url}" user-password jhu_admin "${DRUPAL_DEFAULT_ACCOUNT_PASSWORD}";
```
Comment (bseeger) [inline drupal/rootfs/etc/cont-init.d/99-custom-setup.sh:150]: I wonder if we should pull this in via an environment variable, or maybe that's what you're wondering, @little9, in your question to @jhujasonw about how private this should be.  At the very least changing this here would protect us from random attacks, but this wouldn't help with a directed attack.

**Comment by bseeger** [comment]
I created this [PR](https://github.com/jhu-idc/idc-isle-buildkit/pull/104) to see if it helps with the test issues seen here.

**Comment by little9** [comment]
@bseeger It looks like this is good now. No more CI weirdness!

**Comment by bseeger** [comment]
Looks good to me. Let's get this in and then update the other [PR](https://github.com/jhu-idc/idc-isle-dc/pull/295) to use it.

**Review by bseeger — APPROVED** [review]

**Pull request merged**